src/libs/ReportUtils.js

```diff
@@ -1843,7 +1846,22 @@ function getReportPreviewMessage(report, reportAction = {}, shouldConsiderReceip
     }
 
     const containsNonReimbursable = hasNonReimbursableTransactions(report.reportID);
-    return Localize.translateLocal(containsNonReimbursable ? 'iou.payerSpentAmount' : 'iou.payerOwesAmount', {payer: payerName, amount: formattedAmount});
+
+    if (containsNonReimbursable) {
+        return Localize.translateLocal('iou.payerSpentAmount', {payer: payerName, formattedAmount});
+    }
+
+    if (isPreviewMessageForParentChatReport) {
+        return Localize.translateLocal('iou.payerOwesAmount', {payer: payerName, amount: formattedAmount});
+    }
+
+    let requestorDisplayName = shouldHidePayerName || report.lastActorAccountID === currentUserAccountID ? '' : getDisplayNameForParticipant(report.lastActorAccountID, !isPreviewMessageForParentChatReport);
+    requestorDisplayName = isPreviewMessageForParentChatReport ? requestorDisplayName : `${requestorDisplayName}:`;
```

Comment:
also here looks like it can be wrapped in the euseMemo

Reply:
same